junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSource.java

```diff
@@ -94,4 +94,14 @@
 	@API(status = EXPERIMENTAL, since = "5.5")
 	String emptyValue() default "";
 
+	/**
+	 * A List of Strings that should be interpreted as {@code null}.
+	 *
+	 * <p>Defaults to {@code {}}.
```

Comment:
Is that syntactically correct for Javadoc?

Reply:
I'm not sure what you exactly mean?
The Paragraph-Tag should be fine. It is used everywhere else even in other Projects.

If you mean the `{}` part I also think it's okay. At least the preview in my IDE and the generated HTML looks good.

Reply:
Yes, I was referring to the `{}` part.

Eclipse displays that as `{}}` in generated Javadoc.

But... if the HTML generated by the build renders it as `{}`, then that's fine.